packages/ckeditor5-core/src/editor/editorconfig.ts

```diff
@@ -36,11 +36,8 @@ export interface EditorConfig {
 	context?: Context;
 
 	/**
-	 * ------TODO redirect to proper guide--------
 	 * The list of additional plugins to load along those already available in the
-	 * {@glink getting-started/legacy/installation-methods/predefined-builds editor build}.
-	 * It extends the {@link #plugins `plugins`} configuration.
-	 * -------------------------------------------
+	 * editor. It extends the {@link #plugins `plugins`} configuration.
```

Comment:
DO we actually use this notation for links? First time me seeing it, honestly.

Reply:
Unfortunately I don't know, these notation for links was there before my changes.